mne_neuron/network.py

```diff
@@ -258,6 +260,28 @@ def gid_to_type(self, gid):
             if gid in gids:
                 return gidtype
 
+    def reset_src_event_times(self, seed=None, debug=False, inc_evinput=0.0):
+        """
+        Reset src (source/external) event times
+        evinputinc is an offset for evoked inputs (added to mean start
+        time - e.g. per trial increment)
+        """
+
+        for feed in self.extinput_list:
+            if seed is None:
+                feed.inc_prng(1000)
+            else:
+                feed.set_prng(seed)
+            feed.set_event_times(inc_evinput)  # uses feed.seed
+
+        for k, lfeed in self.ext_list.items():  # dictionary of lists...
+            for feed in lfeed:  # of feeds
+                if seed is None:
+                    feed.inc_prng(1)
+                else:
+                    feed.set_prng(seed)
```

Comment:
do you need this if-else clause? since `seed` is always `None`

Reply:
Agree, this can be removed